scripts/helpers.sh

```diff
@@ -107,6 +102,11 @@ function start {
         # use different command here for staging/production
         pm2 start npm --name "id" -- run start
     fi
+    sleep 10
```

Comment:
this needs to happen before Tonomy ID starts

Reply:
i think it should be after as it will make the bundler 100% sure that the ID app compiles before we try the auto refresh feature.
the sdk gets build when it start and when we install its packages
before we start the id app we start rsync
before i did this the ID app crashed sometimes

Reply:
ok agreed